xbmc/filesystem/test/TestFile.cpp

```diff
@@ -103,7 +103,7 @@ TEST(TestFile, Write)
   char buf[30];
   memset(&buf, 0, sizeof(buf));
 
-  ASSERT_TRUE((file = XBMC_CREATETEMPFILE("")) != NULL);
+  ASSERT_NE(nullptr, file = XBMC_CREATETEMPFILE(""));
```

Comment:
whats nullptr - we normally use NULL - also not sure if extra parantheses are needed around the assignment (depends on the macro i guess)